examples/activation_and_marker_tracking/activation_tracker.py

```diff
@@ -0,0 +1,219 @@
+from scipy.integrate import solve_ivp
```

Comment:
I do not think it is useful to have a completely separated folder for this example. It seems that the only difference is the use of excitation instead of activation. Would just a switch case in the relevant lines have worked? If not, they seem to share the same model, so they could have been in the same folder so it would not need to duplicate the code.

If the run time of this is under 30s, please provide a validated test too